apps/prairielearn/src/sync/syncFromDisk.ts

```diff
@@ -37,35 +34,56 @@ export async function syncDiskToSqlWithLock(
   courseDir: string,
   logger: Logger,
 ): Promise<SyncResults> {
+  async function timed<T>(label: string, fn: () => Promise<T>): Promise<T> {
```

Comment:
Consider lifting this into a library function that takes a longer and returns the function. I feel like we should be using this generically in more server jobs.

Reply:
I'll do this in a future PR when we start using this in another place.